### ISSUE TITLE
selectOptGroup does not render (correctly)

### ISSUE THREAD

**Issue by litrik** [rjaros/kvision#345]
I have noticed that selectOptGroup seems broken in 5.8.0. I'm trying to upgrade from 3.16.1 where everything works fine.

1. Paste the selectOptGroup example from the documentation (https://kvision.gitbook.io/kvision-guide/2.-frontend-development-guide/form-controls-guide) into the Showcase sample (FormTab.kt). Patch file
[selectOptGroup_from_docs_patch.zip](https://github.com/rjaros/kvision/files/7997382/selectOptGroup_from_docs_patch.zip)
2. You'll notice that the opt group does not render:
![2022-02-03_19-59](https://user-images.githubusercontent.com/688765/152410695-3698418f-b8fb-4d5d-9820-9fc4d442f196.png)

Additionally, if you use only optGroups, only the first one renders with the correct options. The next ones get the same options as the first one.

So the following code:

```
selectOptGroup("Option A") {
    selectOption("a1", "A 1", icon = "fab fa-apple")
    selectOption("a2", "A 2", icon = "fab fa-google")
}
selectOptGroup("Option B") {
    selectOption("b1", "B 1", icon = "fab fa-apple")
    selectOption("b2", "B 2", icon = "fab fa-google")
}
````
renders as

![2022-02-03_20-04](https://user-images.githubusercontent.com/688765/152411316-c009a478-026d-4925-bf8b-35be0a4c9c76.png)

**Comment by rjaros** [comment]
Hi. You are right - optgroups are broken in Bootstrap Select v1.14.0-beta2, which is used in KVision 5. See: https://github.com/snapappointments/bootstrap-select/issues/2607 
Unfortunately there is currently no other release. I will try to fork and make my own release for the time being.

**Comment by rjaros** [comment]
Fixed in 5.8.1
